src/meta/src/hummock/manager/mod.rs

```diff
@@ -3225,77 +3226,95 @@ impl HummockManager {
                     match event {
                         RequestEvent::PullTask(PullTask { pull_task_count }) => {
                             assert_ne!(0, pull_task_count);
+
+                            // pull task handler
+                            // 1. pull task from hummock_manager
+                            // 2. send task to compactor
+                            async fn pull_task_handler(hummock_manager: Arc<HummockManager>, pull_task_count: u32, group: u64, selector: &mut Box<dyn CompactionSelector>, task_type: TaskType, compactor: Arc<Compactor>) -> usize {
+                                let mut pick_task_count = 0;
```

Comment:
Is it okay to recount `pick_task_count` for each pull? If there are enough tasks, the count of picked tasks will be 2 times of the target.